Source/non_ip/ai.conf

```diff
@@ -53,3 +53,10 @@ DOMAIN-SUFFIX,clipdrop.co
 DOMAIN-SUFFIX,jasper.ai
 # OpenArt
 DOMAIN-SUFFIX,openart.ai
+# GitHub Copilot
+DOMAIN-SUFFIX,githubcopilot.com
+DOMAIN,copilot-proxy.githubusercontent.com
+# Github Copilot (Claude)
+DOMAIN,copilot-telemetry.githubusercontent.com
```

Comment:
Telemetry domains never have Geo-blocking. Besides the telemetry domain should not be included here in the first place.